javascript/javascript3/week3/shopping-cart/shopping.js

```diff
@@ -0,0 +1,99 @@
+class Product {
+    constructor(name, price) {
+      this.name = name;
+      this.price = price;
+    }
+      convertToCurrency(currency){
+        switch(currency){
+          case "USD": this.price *= 0.16
+          break;
+          case "EUR": this.price *= 0.13
+          break;
+          case "ARS": this.price *= 14,53
+        }
+        return this.price
+
+      }
+  }
+  
+class ShoppingCart {
+    constructor() {
+      this.products = [];
+    }
+  
+    addProduct(product) {
+      shoppingCart.products.push(product)
```

Comment:
inside the class you can use this for all the variable and function. Here instead of shoppingCart.products you should use this.products